sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_component_content_v1/query.sql

```diff
@@ -0,0 +1,83 @@
+-- get the window width by visit, which is needed for rownumber
+WITH visit_window_width AS (
+  SELECT
+    MIN(
+      IF(
+        category = 'newtab'
+        AND name = "opened",
+        SAFE_CAST(mozfun.map.get_key(extra, "window_inner_width") AS INT),
+        NULL
+      )
+    ) AS newtab_window_inner_width,
+    mozfun.map.get_key(extra, 'newtab_visit_id') AS newtab_visit_id
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`,
+    UNNEST(events)
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+  GROUP BY
+    newtab_visit_id
+),
+-- unnest pocket events
+pocket_events_unnested AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id AS client_id,
+    mozfun.map.get_key(extra, 'newtab_visit_id') AS newtab_visit_id,
+    SAFE_CAST(
+      mozfun.norm.browser_version_info(client_info.app_display_version).major_version AS INT64
+    ) AS app_version,
+    normalized_country_code AS country,
+    name AS event_name,
+    extra AS event_details,
+    ping_info
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`,
+    UNNEST(events)
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND category = 'pocket'
+)
+-- aggregate metrics to the visit_id level
+SELECT
+  submission_date,
+  sample_id,
+  country,
+  app_version,
+  client_id,
+  newtab_visit_id,
+  newtab_window_inner_width,
```

Comment:
> Idea was to include only tiles specific to the component interaction, and any additional metadata about the visit can be obtained from the visit-level table.

The read from the [Description](https://github.com/mozilla/bigquery-etl/pull/7988#issue-3346786592) appears to suggest that non-component specific metadata is going to be obtained from the newtab visit table. This makes sense because duplicating the non-component level fields causes unnecessary redundancy.
To that effect, perhaps the only additional fields needed would be `newtab_visit_id` and `submission_date` (those needed for the join with the visits table)?

Reply:
I wanted to try to keep the creation of the view/table separate from `newtab_visits_daily` in case we take action on the proposal to populate `newtab_visits_daily_v1` with the component table, I didn't want to create a circular dependency

Reply:
> I wanted to try to keep the creation of the view/table separate from `newtab_visits_daily` in case we take action on the proposal to populate `newtab_visits_daily_v1` with the component table, I didn't want to create a circular dependency

What you are thinking involves a major design change and may involve changing what you may be doing here. Therefore, not relevant to this PR

Reply:
That's fair, I'd still prefer to keep them independent for now.  Are there serious downsides I might be missing?  I know it is less memory efficient, but is it so much so that this is inadvisable?

Reply:
Oh I meant I wanted to keep `newtab_visits_daily` and this table independent, so I'd like to keep `newtab_window_inner_width` and `app_version` for this reason.

I wanted `sample_id` because sometimes it's useful to be able to reduce the size, and `country` is one of the most common to filter on but it could probably be removed. `experiments` is another one that seemed like it'd be used commonly enough that it'd be worth keeping.  @ilanasegall do you have opinions on which fields to keep, trying to balance making this table easy to use and not including too many fields that could be obtained by joining to the visits table.?

Reply:
If this is meant to be a quick PR to serve your and @ilanasegall immediate use cases, then this PR perhaps does not need to go through an elaborate review process.

I would however recommend that a proper data validation done before its deployed and used in any mainstream applications

Reply:
Happy to take feedback on what you think the proper approach to data validation is here.  The [notebook](https://colab.research.google.com/drive/1Xk_oo5Zc50tNzh-g7V7HOeC2dFNABJa8) I created was what I've done so far.  Because there is so little processing and no existing source of truth to compare against, checking the handling of "edge cases" seemed like a reasonable approach.  I also check that the queries run and looked over the results to ensure there wasn't any obvious errors.

Let me know what else you'd like to see done, I'd rather spend a bit more time here and get it right the first time than have to do a follow-up PR to fix avoidable mistakes!

Reply:
> Happy to take feedback on what you think the proper approach to data validation is here.

The visit level table that we did in H1, Nadia had done an extensive data validation which I believe she also documented somewhere to share with other DS folks. Since you are also going to be the consumer if the data, I would let you decide what kind of validation you would like to perform. Would also be good to share with other DS folks for their approval.